client/jetpack-cloud/sections/partner-portal/issue-license-form/style.scss

```diff
@@ -25,3 +26,34 @@
 		}
 	}
 }
+
+.issue-license-form__top {
+	margin-bottom: 1rem;
+	margin-top: -1.5rem;
+}
+
+.issue-license-form__bottom {
+	display: flex;
+	flex-wrap: wrap;
+}
+
+p.issue-license-form__description {
+	margin-bottom: 0;
+	align-self: flex-end;
+	font-size: 0.875rem;
+	color: #333;
+}
+	
+@media ( max-width: 960px ) {
```

Comment:
Some of the code uses the Gutenberg breakpoints, which has `960px` as an option:
```
@import '@wordpress/base-styles/_breakpoints.scss';
@import '@wordpress/base-styles/_mixins.scss';
```

Should we stick with how the rest does it or do we differ in some way where we don't want to follow them (just because they exist as a variable/mixin)?

Reply:
Again, this doesn't work as expected, and I'm not sure why. I followed the existing pattern. I'd be more concerned if there was a high likelihood that agencies would be working from mobile devices, but I doubt it's a big issue.